_posts/2021-09-07-ols-4-annoucement.md

```diff
@@ -0,0 +1,123 @@
+        {% capture prev-mentors %}{{ prev-mentors }}, {{ m }}{% endcapture %}
+    {% endfor %}
+{% endfor %}
+
+{% assign all-participants = '' %}
+{% assign all-p-countries = '' %}
+{% assign all-mentors = '' %}
+{% assign all-m-countries = '' %}
+{% assign all-keywords = '' %}
+{% assign prev-part-count = 0 %}
+{% assign prev-mentor-count = 0 %}
+
+{% for project in projects %}
+
+<!-- parse participants of the project -->
+    {% for p in project.participants %}
+<!-- for name and link to them -->
+        {% capture all-participants %}{{ all-participants }}, [{{ people[p].first-name }} {{ people[p].last-name }}](/{{ cohort }}/projects-participants#{{ p }}){% endcapture %}
+<!-- for list of countries -->
+        {% if people[p].country %}
+            {% capture all-p-countries %}{{ all-p-countries }}, {{ people[p].country }}{% endcapture %}
+        {% endif %}
+    {% endfor %}
+
+<!-- parse mentors of the project -->
+    {% for m in project.mentors %}
+<!-- for name and link to them -->
+        {% capture all-mentors %}{{ all-mentors }}, [{{ people[m].first-name }} {{ people[m].last-name }}](/{{ cohort }}#{{ m }}){% endcapture %}
+<!-- for list of countries -->
+        {% if people[m].country %}
+            {% capture all-m-countries %}{{ all-m-countries }}, {{ people[m].country }}{% endcapture %}
+        {% endif %}
+<!-- add +1 if participant in previous cohort -->
+        {% if prev-participants contains m %}
+            {% assign prev-part-count = prev-part-count | plus: 1 %}
+        {% endif %}
+<!-- add +1 if mentor in previous cohort -->
+        {% if prev-mentors contains m %}
+            {% assign prev-mentor-count = prev-mentor-count | plus: 1 %}
+        {% endif %}
+    {% endfor %}
+
+<!-- parse keywords -->
+    {% for k in project.keywords %}
+        {% capture all-keywords %}{{ all-keywords }}, {{ k }}{% endcapture %}
+    {% endfor %}
+{% endfor %}
+
+<!-- transform into lists -->
+{% assign p-participants = all-participants | remove_first: ', ' | split: ", " | uniq | sort %}
+{% assign p-countries = all-p-countries | remove_first: ', ' | split: ", " | uniq | sort %}
+{% assign p-mentors = all-mentors | remove_first: ', ' | split: ", " | uniq | sort %}
+{% assign m-countries = all-m-countries | remove_first: ', ' | split: ", " | uniq | sort %}
+
+We are thrilled to announce that [{{ p-participants | size }} members](/{{ cohort }}/projects-participants/#participants), who are the project leads of [{{ projects | size }} diverse projects](/{{ cohort }}/projects-participants/#projects), have joined the fourth cohort of the Open Life Science mentoring program - OLS-4!
+
+### Meet our mentees!
+
+The mentees joining this program are {{ p-participants | join: ', ' }}. These individuals are based in {{ p-countries | size }} countries ({{ p-countries | join: ', ' }}) where they will be leading their respective projects. 
+
+Topics for their projects include {{ keywords | join: ', ' }}.
```

Comment:
we need to add some keywords there as I did have them in the project description sheet

Reply:
added some keywords.

Reply:
Thanks. I fixed sth above that should make them displayed